src/sci/asci.h

```diff
@@ -77,19 +78,16 @@ class ASCI : public ActiveSpaceMethod {
 
     // ==> Class Interface <==
 
-    /// Compute the energy
-    double compute_energy() override;
-
     /// Compute the reduced density matrices up to a given particle rank (max_rdm_level)
     std::vector<RDMs> rdms(const std::vector<std::pair<size_t, size_t>>& root_list,
```

Comment:
This function is a member function of `ExcitedStateSolver` and should not appear here.